draft-private-access-tokens.md

```diff
@@ -644,14 +647,10 @@ operation:
 
 - RSA Blind Signatures {{BLINDSIG}}, for issuing and constructing Tokens as described in {{redemption}}.
 
-- Prime Order Groups (POGs), for computing stable mappings between (Client, Origin) pairs. This
-  document uses notation described in {{!VOPRF=I-D.irtf-cfrg-voprf, Section 2.1}}, and, in particular,
-  the functions RandomScalar(), Generator(), SerializeScalar(), SerializeElement(), and HashToScalar().
-
-- Non-Interactive proof-of-knowledge (POK), as described in {{nizk-dl}}, for verifying correctness of Client requests.
+- Schnorr signatures, as described in {{!RFC8032}}, for verifying correctness of Client requests.
```

Comment:
Is this still Schnorr? Or is it ed25519?

Reply:
They are the same, but I'll fix =)